presto-hive/src/test/java/com/facebook/presto/hive/metastore/InMemoryHiveMetastore.java

```diff
@@ -395,6 +395,17 @@ public void grantTablePrivileges(String databaseName, String tableName, String g
     }
 
     @Override
+    public void revokeTablePrivileges(String databaseName, String tableName, String grantee, Set<PrivilegeGrantInfo> privilegeGrantInfoSet)
+    {
+        Set<HivePrivilege> hivePrivileges = privilegeGrantInfoSet.stream()
+                .map(HivePrivilege::parsePrivilege)
+                .flatMap(Collection::stream)
+                .collect(toImmutableSet());
+
+        setTablePrivileges(grantee, USER, databaseName, tableName, hivePrivileges);
```

Comment:
what does setTablePrivileges do? Will it update the table privileges after removing the ones we want to revoke?